src/views/studio/studio-comments.jsx

```diff
@@ -127,12 +121,9 @@ const StudioComments = ({
                         <TopLevelComment
                             hasThreadLimit
                             author={comment.author}
-                            canDelete={canDeleteAnyComment ||
-                                (canDeleteOwnComment && comment.author.username === username)}
                             canDeleteWithoutConfirm={canDeleteCommentWithoutConfirm}
-                            canReply={shouldShowCommentComposer}
-                            canReport={canReportComment}
                             canRestore={canRestoreComment}
+                            commentComponent={StudioComment}
```

Comment:
So basically this prop is kind of like us passing in a constructor for how the TopLevelComments component should structure its reply comments?

Reply:
Yeah I guess you could say that. Another way that would be a bit more explicit and require less knowledge of the internals of TopLevelComment would be to use a render prop, where the consumer of TopLevelComment would provide a child that was a function called with the props that would be passed to component. That way the TopLevelComment consumer could more explicitly customize which props (as opposed to right now where we are implicitly passing a set of props that can be overridden/intercepted).